src/main/java/io/jenkins/plugins/orka/client/DeploymentRequest.java

```diff
@@ -59,9 +62,35 @@ public DeploymentRequest(String vmConfig, String name, String node, String sched
         this.shouldGenerateName = StringUtils.isNotBlank(this.name);
     }
 
+    public DeploymentRequest(String vmConfig, String name, String image, Integer cpu, String memory, String node,
```

Comment:
Same comment about less specific methods callign more specific ones instead of repeating the code